docs/elasticsearch.md

```diff
@@ -14,195 +14,436 @@ sidebar_label: Elasticsearch
+elastic-coordinator-0   1/1     Running   0          31s
+elastic-data-0          1/1     Running   0          56s
+elastic-data-1          1/1     Running   0          44s
+elastic-master-0        1/1     Running   0          2m31s
+elastic-master-1        1/1     Running   0          119s
+elastic-master-2        1/1     Running   0          90s
+```
 
-It is not seamless to increase the cStor volume size (refer to the roadmap item). Hence, it is recommended that sufficient size is allocated during the initial configuration. 
+#### Verifying Services
 
-**Monitor cStor Pool size**
+```bash
+kubectl get svc -n $namespace_name
+```
+```
 
-As in most cases, cStor pool may not be dedicated to just Elasticsearch database alone. It is recommended to watch the pool capacity and add more disks to the pool before it hits 80% threshold. See [cStorPool metrics](/docs/next/ugcstor.html#monitor-pool). 
+NAME                     TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
+elastic-coordinator-hs   ClusterIP   None         <none>        9200/TCP   62s
+elastic-data-hs          ClusterIP   None         <none>        9200/TCP   87s
+elastic-ingest-hs        ClusterIP   None         <none>        9200/TCP   50s
+elastic-master-hs        ClusterIP   None         <none>        9200/TCP   3m2s
+kubernetes               ClusterIP   10.48.0.1    <none>        443/TCP    5h18m
+```
 
 
+#### Verifying Elastic instance status
 
-<br>
+```bash
+kubectl kudo plan status --namespace=$namespace_name \
+ --instance $instance_name
+```
+```
 
-<hr>
+Plan(s) for "elastic" in namespace "default":
+.
+└── elastic (Operator-Version: "elastic-7.0.0-0.2.1" Active-Plan: "deploy")
+    └── Plan deploy (serial strategy) [COMPLETE], last updated 2021-02-22 16:18:17
+        ├── Phase deploy-master (parallel strategy) [COMPLETE]
+        │   └── Step deploy-master [COMPLETE]
+        ├── Phase deploy-data (parallel strategy) [COMPLETE]
+        │   └── Step deploy-data [COMPLETE]
+        ├── Phase deploy-coordinator (parallel strategy) [COMPLETE]
+        │   └── Step deploy-coordinator [COMPLETE]
+        └── Phase deploy-ingest (parallel strategy) [COMPLETE]
+            └── Step deploy-ingest [COMPLETE]
+```
 
-<br>
+#### Accessing Elasticsearch
 
+Enter into one of the master pod using exec command:
+```bash
+kubectl exec -it elastic-master-0 -- bash
+```
+```bash
 
+[root@elastic-master-0 elasticsearch]#
+```
 
+Run below command inside Elastic master pod:
+```bash
+curl -X POST "elastic-coordinator-hs:9200/twitter/_doc/" -H 'Content-Type: application/json' -d'
+ {
+     "user" : "openebs",
+     "post_date" : "2021-03-02T14:12:12",
+     "message" : "Test data entry"
+ }
+ '
+```
+```json
 
+{"_index":"twitter","_type":"_doc","_id":"LoliyXcBg9iVzVnOj5QL","_version":1,"result":"created","_shards":{"total":2,"successful":1,"failed":0},"_seq_no":0,"_primary_term":1}
+[root@elastic-master-0 elasticsearch]#
+```
 
-## Configuration details
+Get results for the above test:
```

Comment:
The output of the above command is lengthy and hence removed . 
The next command is demonstration of accessing elasticsearch using the localhost and this was used in the earlier documentation and hence continued to use in this doc also

Reply:
ok. If output is lengthy, we can mention that this output shows something something. So that users will know that why this command is required to run.
Also, if we are getting the elasticsearch details by curling localhost:9200, does this means installation is successful? If so, we can mention that if the command is running successful, then it will show that your ElastiSearch application is successfully deployed. just a thought.